crates/cargo-wdk/src/actions/package/mod.rs

```diff
@@ -0,0 +1,370 @@
+#[cfg(test)]
+mod tests;
```

Comment:
Add module comments. We probably don't need them for small modules, but they do help with bigger ones.